build.rs

```diff
@@ -0,0 +1,91 @@
+use std::process::Command;
+use std::path::Path;
+use std::str;
+use std::fs::File;
+use std::io::Write;
+
+const DEP_LICENSES_PATH: &str = "target/DEP_LICENSES";
+const IF_DEBUG_TEXT: &str = "This is a debug build. \
+    It should not be used by users. \
+    Please obtain a release build instead.";
+
+
+// The build script for `sic` primarily makes licenses of dependencies available to the installed
+// executable.
+// Previously we used cargo-make for this purpose, but in case `sic` is installed by running
+// `cargo install --force sic`, `cargo make release` is not invoked.
+// To fix that, we will now use this build script instead.
+// As a bonus, it should now work both on Windows and Linux out of the box; i.e. on Windows it
+// doesn't rely on some installed tools like which anymore.
+fn main() {
+    println!("Starting the pre-processing of a `sic` build.");
+
+    // If we are not creating a release; do not complicate the build.
+    if cfg!(debug_assertions) {
+        write(DEP_LICENSES_PATH, IF_DEBUG_TEXT.as_bytes());
+        return;
+    }
+
+    // Check if cargo-bom is available in our PATH.
+    let cargo_bom_might_be_installed = if cfg!(windows) {
+        Command::new("where.exe")
+            .args(&["cargo-bom"])
+            .output()
+            .expect("`where.exe` unavailable.")
+            .stdout
+    }
+    else {
+        Command::new("which")
+            .args(&["cargo-bom"])
+            .output()
+            .expect("`which` unavailable.")
+            .stdout
+    };
+
+    // Convert to str
+    let str_path = str::from_utf8(cargo_bom_might_be_installed.as_slice())
+       .expect("Unable to convert path.")
+       .trim();
+
+    // Convert to a path and check if it exists;
+    // If it does; cargo-bom has been found and doesn't need to be installed first.
+    let path = Path::new(str_path);
+
+    println!("-> {:?}", path);
```

Comment:
Perhaps, this println to crate build output should be prefixed with text like "cargo-bom path found at: " instead of "->".
Additionally, it might be useful to move it below `!path.exists()`.